autoload/leaderf/python/leaderf/gtagsExpl.py

```diff
@@ -395,10 +395,15 @@ def _isVersionControl(self, filename):
             return False
 
     def _generateDbpath(self, path):
+        with_gutentags = lfEval("get(g:, 'Lf_GtagsGutentags', 0)")
+        sep_char = '-' if with_gutentags else '_'
         if os.name == 'nt':
-            db_folder = re.sub(r'[\\/]', '_', path.replace(':\\', '_', 1))
+            if with_gutentags:
```

Comment:
The return value of `lfEval("get(g:, 'Lf_GtagsGutentags', 0)")` is a str, it's '0' or '1', so '0' would always be `True`.
The else branch would never be executed.

Reply:
Thank you. I have fixed the problem, please check it again.